releasenotes/notes/add-hexagonal-lattice-couplingmap-d3b65b146b6cd1d1.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Add a new constructor method, :meth: `~qiskit.transpiler.CouplingMap.from_hexagonal_lattice`, to
+    the :class:`~qiskit.transpiler.CouplingMap` class.
+
+    For example:
+
+    .. jupyter-execute::
+      from qiskit.transpiler import CouplingMap
+      cmap = CouplingMap.from_hexagonal_lattice(2, 2)
+      cmap.draw()
```

This should fix the docs error:

```suggestion
    .. jupyter-execute::

      from qiskit.transpiler import CouplingMap
      cmap = CouplingMap.from_hexagonal_lattice(2, 2)
      cmap.draw()
```